Source/Dafny/DafnyOptions.cs

```diff
@@ -158,6 +158,11 @@ public enum IncludesModes {
     protected override bool ParseOption(string name, Bpl.CommandLineOptionEngine.CommandLineParseState ps) {
       var args = ps.args; // convenient synonym
       switch (name) {
+        case "OneAssertAtATime":
+        case "oneAssertAtATime":
```

Comment:
I'd rather not add this yet, it's premature. The main use case for passing `/vcsSplitOnEveryAssert` on the command line (as opposed to the `{:vcs_split_on_every_assert}` attribute) is to get data on individual assertions, and we don't yet provide a way to map anonymous Boogie split results back to the single Dafny assertion that created them (although https://github.com/boogie-org/boogie/pull/505 is a big step towards that). So I'm happier leaving this as a low-level Boogie option, rather than starting to sell it as a well-encapsulated Dafny option.

Reply:
I did not know about the _attribute_ `{:vcs_split_on_every_assert}`, I'll document it because this is useful !
I'll remove the reference to `/oneAssertAtATime`.
> we don't yet provide a way to map anonymous Boogie split results back to the single Dafny assertion that created them
I'm working on that on another PR, which will need to point to this documentation (it needs a valid URL)
I think it is fine to describe this option for now, because still it could help someone get out of something that does not verify.